docs/reference/ccr/apis/follow/get-follow-stats.asciidoc

```diff
@@ -75,6 +75,9 @@ task. In this situation, the following task must be resumed manually with the
 `index`::
 (string) The name of the follower index.
 
+`total_global_checkpoint_lag`::
+(long) Total number of pending indexing operations
```

Comment:
```suggestion
(long) Indication of how much the follower is lagging the leader. This is the sum of the difference between the `leader_global_checkpoint` and the `follower_global_checkpoint` for all shards.
```

I believe this is better because (a) it keeps in line with how `follower_global_checkpoint` is explained below, and (b) we do not only replicate "indexing" operations.